db/migrate/20240219122800_populate_needed_items.rb

```diff
@@ -0,0 +1,7 @@
+class PopulateNeededItems < ActiveRecord::Migration[7.0]
+  def change
+    Partners::Child.find_each do |child|
+      child.needed_item_ids = [child.item_needed_diaperid] if child.item_needed_diaperid.present?
```

Comment:
Double checking - this actually saves a record to the `needed_items` table, right?